tools/install.py

```diff
@@ -120,6 +121,17 @@ def corepack_files(action):
 #   'pnpx': 'dist/pnpx.js',
   })
 
+  # On z/OS, we install node-gyp for convenience as native add-ons
+  # are dependend on
```

Comment:
Can you expand on why this is needed on z/OS? The regular developer flows for installing modules normally have `npm` automatically run `node-gyp` from its own dependencies either automatically on `npm install` (if no install script was defined and a gyp file found) or by the individual modules scripting a call to it as a devdependency.

Reply:
It is because some vendors don't have external access and may want to build their own native node.js modules